src/s1ap/s1apContextManager/s1apContextWrapper.cpp

```diff
@@ -71,6 +75,19 @@ bool getControlBlockDetailsEnbFd_cpp(uint32_t enbFd, struct EnbStruct *enbStruct
         EnbContext* enbCtxt_p = static_cast <EnbContext *>(cb->getPermDataBlock());
         enbStructCtx->tai_m.tac = enbCtxt_p->getTai().tac; 
         strcpy(enbStructCtx->eNbName, enbCtxt_p->getEnbname());
+        int val = mme::S1apDataGroupManager::Instance()->deleteenbIdkey(
```

Comment:
May be good time to change function name..since initially getControlBlockDetailsEnbFd_cpp was used to get details,  but now its used for deleting/cleanup

Reply:
done